packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionScreen.tsx

```diff
@@ -31,10 +31,6 @@ const JurisdictionScreen = ({ setIsCheckBoxChecked }: TJurisdictionScreenProps)
         [marketType, mt5AccountsList]
     );
 
-    useEffect(() => {
-        return () => setCfdState({ selectedJurisdiction: '' });
-    }, [setCfdState, setIsCheckBoxChecked]);
```

Comment:
this useEffect has caused the payload to be `company: " "` 

I have added a closeHandler to unset the selectedJurisdiction in `JurisdictionModal.tsx`